src/app/components/PlayerPanel.ts

```diff
@@ -0,0 +1,132 @@
+import { Player } from '../../Player';
+import { Render } from '../utils/Render';
+import { View } from '../View';
+import { flatten } from 'lodash';
+import { AnimalNames } from '../../Enums/AnimalNamesEnum';
+
+export class PlayerPanel {
+  /**
+   * Creates PlayerPanel based on data given
+   * @param player accepts instance of current player
+   * @param view accepts instance of View componnet
+   */
+  constructor(private player: Player, private view: View) {}
+
+  createPlayerPanel(): HTMLElement {
+    return Render.elementFactory(
+      'div',
+      {
+        className: 'player-panel',
+        style: `background-color: ${this.player.theColor};`,
+      },
+      this.createPlayerBoard(),
+      this.createResultWindow(),
+      this.createButtonPanel(),
+    );
+  }
+
+  private createPlayerBoard(): HTMLElement {
```

Comment:
Do we not have this method as a class PlayerBoard? Is there a way to use it instead of creating an extra method?

Reply:
after PlayerBoard will be finished, I will adjust method or remove it